src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/lettertoprison/LetterToPrisonContextFactoryTest.kt

```diff
@@ -53,13 +54,21 @@ class LetterToPrisonContextFactoryTest {
       lastReleasePrison = PrisonId("BOB"),
       licenceNameCategory = NameFormatCategory.FIRST_LAST,
       recallLength = recallLength,
+      bookingNumber = "ABC123",
+      lastReleaseDate = LocalDate.now(),
```

Comment:
How about local vals for random values of these booleans, enums and dates, so that the assert is then more convincing?